addons/mail/static/src/discuss/core/common/channel_invitation.js

```diff
@@ -11,7 +11,7 @@ export class ChannelInvitation extends Component {
     static components = { ImStatus, ActionPanel };
     static defaultProps = { hasSizeConstraints: false };
     static props = ["hasSizeConstraints?", "thread", "close", "className?"];
-    static template = "discuss.ChannelInvitation";
+    static template = "mail.discuss.ChannelInvitation";
```

Comment:
Hello, I don't think we're supposed to change template name in stable.

Also we don't want this prefix in this case, as we still want some day to split the modules and the separate prefix was a first stepping stone.